app/controllers/likes_controller.rb

```diff
@@ -27,6 +27,8 @@ def create
       if @like.save
         format.html { redirect_back fallback_location: root_url, notice: "Like was successfully created." }
         format.json { render :show, status: :created, location: @like }
+        format.js
+
```

Comment:
nitpik: please be mindful of unnecessary empty white spaces and delete when you can!